common/appointments/participants.ts

```diff
@@ -0,0 +1,14 @@
+import { Lecture } from '../entity/Lecture';
+import { prisma } from '../prisma';
+import { User } from '../user';
+
+export async function isAppointmentParticipant(lecture: Lecture, user: User): Promise<boolean> {
+    return !!(await prisma.lecture.findFirst({
+        where: {
```

Comment:
We could select: { id: true } to reduce the amount of data returned from the db

Reply:
Nice one! 👍🏻

Reply:
Yeah one day I request a .exists({ where: { ... } }) feature from Prisma